### ISSUE TITLE
example site should be faster, and with improved accessibility

### ISSUE THREAD

**Issue by dna113p** [headwirecom/peregrine-cms#56]
To properly show Peregrine as a fast and accessible CMS, we need to make sure that our examples demonstrate that well.

For a starting point, we should get our Google Lighthouse audit scores for mobile devices on 3G up to 100%. For the example vue site: 
https://googlechrome.github.io/lighthouse/viewer/?gist=0301c3017982a131b2a03007b71e1cdc

**Comment by reusr1** [comment]
I think this issue can be closed for now as themecleanflex together with mod_pagespeed provides this
